<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartBoard</class>
 <widget class="QMainWindow" name="SmartBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Smart Board</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="optionsAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuBoard">
    <property name="title">
     <string>Board</string>
    </property>
    <addaction name="publishAction"/>
    <addaction name="searchAction"/>
    <addaction name="subscribeBoardAction"/>
    <addaction name="unsubscribeAction"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="helpAction"/>
    <addaction name="aboutSBAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuBoard"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="optionsAction">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="publishAction">
   <property name="text">
    <string>Publish announsment</string>
   </property>
  </action>
  <action name="searchAction">
   <property name="text">
    <string>Search information</string>
   </property>
  </action>
  <action name="aboutSBAction">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="helpAction">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="subscribeBoardAction">
   <property name="text">
    <string>Subscribe board</string>
   </property>
  </action>
  <action name="unsubscribeAction">
   <property name="text">
    <string>Unsibscribe board</string>
   </property>
  </action>
  <action name="viewAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View Board</string>
   </property>
   <property name="toolTip">
    <string>View Board</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
